### ISSUE TITLE
`<xf:select1>` and `<xf:select>` with `appearance="full"` to have `aria-required` and `aria-invalid`

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#6302]
Similar to #6279 (but this is not an XBL component).

[+1 from customer](https://3.basecamp.com/3600924/buckets/1956846/messages/7175826989)

**Comment by ebruchez** [comment]
The idea is to put these attributes on the element:

```html
<span
    class="xforms-items xforms-help-popover-control"
    role="radiogroup"
    tabindex="0"
    aria-labelledby="section-1-section≡grid-1-grid≡control-2-control≡≡l"
    aria-describedby="section-1-section≡grid-1-grid≡control-2-control≡≡t"
    aria-errormessage="section-1-section≡grid-1-grid≡control-2-control≡≡a"
    aria-required="true">
```

Now, this element doesn't have an id. It will need one to output updates to the client.

**Comment by ebruchez** [comment]
Also cherry-picked to 2022.1 customer branch.
